configs/localization/ssn/ssn_r50_450e_thumos14_rgb_test.py

```diff
@@ -0,0 +1,187 @@
+# model training and testing settings
```

Comment:
ssn_r50_1x5_450e_thumos14_rgb_test

Reply:
unused part in the config file can be removed

Reply:
Since SSN uses different STPP at training and testing, we have to use two configs. But those two configs can share the same checkpoint in same working directory './work_dirs/ssn_r50_450e_thumos14_rgb'.